src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -722,6 +723,18 @@ for unordered lists.
 - Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dictum a massa
   sit amet ullamcorper.
 
+## Links (external)
+
+```html
+Lorem ipsum dolor sit amet,
+<a href="example.com" rel="noopener">consectetur adipiscing elit</a>.
```

Comment:
This example isn't using `target="_blank"`

Reply:
I think css works if it's using `rel=noopener` or `target=_blank`.